ocs2_ros_interfaces/src/command/TargetTrajectoriesKeyboardPublisher.cpp

```diff
@@ -84,6 +93,48 @@ void TargetTrajectoriesKeyboardPublisher::publishKeyboardCommand(const std::stri
   }  // end of while loop
 }
 
+/******************************************************************************************************/
+/******************************************************************************************************/
+/******************************************************************************************************/
+void TargetTrajectoriesKeyboardPublisher::publishJoystickCommand(const std::string& commadMsg) {
+  while (rclcpp::ok()) {
+    std::cout << commadMsg << ": ";
+    std::this_thread::sleep_for(5s);
+    // getCommandLine();
+    // vector_t commandLineInput = getCommandLine().cwiseMin(targetCommandLimits_).cwiseMax(-targetCommandLimits_);
```

Comment:
@kylc @senceryazici if I don't call this function getCommandLine() 
then the process dies for some reason!